api/pom.xml

```diff
@@ -239,13 +239,13 @@
                 <version>2.8</version>
                 <configuration>
 		    <additionalparam>-Xdoclint:none</additionalparam>
-                    <doctitle>Concurrency Utilities for Java EE ${project.version} API Specification</doctitle>
+                    <doctitle>Jakarta EE Concurrency ${project.version} API Specification</doctitle>
                     <bottom>
 <![CDATA[Copyright &#169; 1996-2013,
     <a href="http://www.oracle.com">Oracle</a>
     and/or its affiliates. All Rights Reserved.
     Use is subject to
-    <a href="{@docRoot}/doc-files/speclicense.html" target="_top">license terms</a>.
+    <a href="https://www.eclipse.org/legal/efsl.php" target="_top">license terms</a>.
```

Comment:
The license text needs to be included with the javadocs, not referenced on the web.
https://www.eclipse.org/projects/handbook/#legaldoc-distribution
The contents of the speclicense.html file should be replaced with the contents of
[this file](https://raw.githubusercontent.com/eclipse-ee4j/jakartaee-api/master/licenses/EFSL.html) for a final release.  For a non-final release, use the contents of [this file](https://raw.githubusercontent.com/eclipse-ee4j/jakartaee-api/master/licenses/EPL.html).

Reply:
do you know where the license file has to go in the javadocs and how to package it? In the api jar it will be in META-INF.

Reply:
I put it in src/main/java/doc-files/speclicense.html.

The API jar file needs to have the EPL license in META-INF/LICENSE.md.